junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/AbstractExtensionContext.java

```diff
@@ -127,4 +130,21 @@ public Store getStore(Namespace namespace) {
 	public <V> Optional<V> getConfigurationParameter(String key, Function<String, V> transformer) {
 		return this.configuration.getRawConfigurationParameter(key, transformer);
 	}
+
+	@Override
+	public ExecutionMode getExecutionMode() {
+		return toJupiterExecutionMode(getPlatformExecutionMode());
+	}
+
+	protected abstract Node.ExecutionMode getPlatformExecutionMode();
+
+	private ExecutionMode toJupiterExecutionMode(Node.ExecutionMode mode) {
```

Comment:
It's the reverse of this: https://github.com/junit-team/junit5/blob/main/junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/JupiterTestDescriptor.java#L171

Should we perhaps move both somewhere, or leave it here?